### DISCUSSION TITLE
Bottom-up vs Top-down, come nasce un libro?

### DISCUSSION THREAD

**Discussion by francescobianco** [Il-Libro-Open-Source/book#6] [Salotto]
Volevo annotare in questa discussione, un dilemma che potrebbe interessarci, soprattutto in questa fase di creazione del board degli ambassador.

Immaginiamo due distinti approcci quello di tipo "Top-down" cioè fissato un indice di capitoli, poi si procede alla stesura del libro attraverso la stesura degli stessi capitoli. Oppure il suo contrario "Bottom-up", si comincia a scrivere dei contenuti basati su dei topic attorno al tema del libro e poi pian-piano si ricostruisce l'alberatura e la struttura dello stesso.

Chiaramente protrebbero apparire come due estremi, e magari l'ottimo sta nel cercare di navigare a vista attorno alle due strategie. Ma volevo condividervi questa questione e sentire le vostre opinioni.

**Comment by Cadienvan** [comment]
Ottimo punto!
Ne parleremo sicuramente nei prossimi giorni, diciamo che ora siamo un po' "col freno tirato" viste le ferie in arrivo, ma sono d'accordo sul trovare la giusta strategia!

**Comment by JeyDi** [comment]
Io proporrei un Top-down, discutiamo di quelle che possono essere le sezioni, mettiamo giù i vari capitoli con una breve descrizione del contenuto e assegnamo i vari capitoli alle persone in base alle preferenze.

Sarebbe da indicare anche una deadline o darci delle scadenze per evitare di andare all’infinito :) Sia per i singoli capitoli che anche per il progetto in generale.

Potrebbe essere interessante fare una sorta di peer-review, ovvero una persona non assegnata ad un particolare capitolo può essere nominata come revisore del contenuto, lasciare commenti, suggerimenti feedback.

Un po’ come funziona nel mondo delle pubblicazioni scientifiche insomma :)

Anche perchè non so voi, ma io non sono proprio espertissimo di tutti i topic che avete menzionato in un’altra discussione :)

**Reply by Cadienvan** [reply]
L'idea è che ogni Ambassador abbia responsabilità di uno o più capitoli e che, di conseguenza, ne porti avanti lo sviluppo coinvolgendo quante più persone possibili, anche altri ambassador, il che si avvicina un po' alla tua idea!

A Settembre ci "smisteremo" peggio delle casate di Hogwarts

**Comment by guizzo** [comment]
@Cadienvan Posso chiudere questa discussion? 

L'ultimo aggiornamento peraltro risale all'8 Agosto, secondo me possiamo archiviarla, che ne pensi?
